source/mainnet/net/guides/run-node-ubuntu.rst

```diff
@@ -74,6 +74,23 @@ The ``concordium-mainnet-node`` service that you just installed will be running
 .. Note::
    If you want more detailed information about building and maintaining a node, or if your node is not running, see the `Building .deb packages for ubuntu distributions README from Concordium <https://github.com/Concordium/concordium-node/blob/main/scripts/distribution/ubuntu-packages/README.md>`__
 
+Enabling inbound connections
+============================
+
+If you are running your node behind a firewall, or behind your home
+router, then you will probably only be able to connect to other nodes,
+but other nodes will not be able to initiate connections to your node.
+This is perfectly fine, and your node will fully participate in the
+Concordium network. It will be able to send transactions and,
+:ref:`if so configured<become-a-baker>`, to bake and finalize.
+
+However you can also make your node an even better network participant
+by enabling inbound connections. By default, ``concordium-node`` listens
+on port ``8888`` for inbound connections. Depending on your network and
```

Comment:
The ubuntu mainnet node by default listens on 8888, but the testnet node by default listens on 8889. 

I believe the same applies to Windows and Mac nodes. The ports have to be different so both can run at the same time.

Reply:
I need confirmation on the Windows and Mac. Who should I ask? Or when you say "I believe" is it not just a belief but a fact?

Reply:
I checked again, and it is indeed as I wrote also for WIn & Mac.